local-cli/runAndroid/runAndroid.js

```diff
@@ -98,13 +93,32 @@ function buildAndRun(args) {
         args.flavor[0].toUpperCase() + args.flavor.slice(1)
       );
     } else {
-      gradleArgs.push('installDebug');
+      gradleArgs.push('install');
```

Comment:
Sure, done. With the `push -f` I removed your commit, sorry for that. I will fix the type in the commit.